Update README.md (#80)
Fabriquartz/ember-railio-form-components: README.md

## Using your own components

You could use your own components by implementing it in the form-field component like above in the block-form usage.

> # Paper-form-field

Reviewer: I would prefer to have the paper-form-fields above using your own component

> Paper-form-field is a variant of the default form-field, using ember-paper for Google Material Design like form-fields. The following sections descripe how the paper-form-field work and what options you can pass to it.

Reviewer: Split to multiple lines to make it more readable in code (mentioned the max-line length for every other file?)

## Usage
```handlebars
{{paper-form-field type="text-area"
                   object=movie
                   propertyPath="description"
                   options=options
                   updated=(action "update")
                   disabled=locked}}
```

## Options
Options is an object the can contain attributes and methods. Every attribute and method will be added to the component. This can be used to reach/supply build in abillities or add/overwrite event handlers.

## htmlAttributes

> If you want to pass attributes to the actual html element that the form field renders, you can add 'htmlAttributes' to the options object. This object can contain thing like placeholder, name of maxlength. You can read the ember-paper documentation for more information.

Reviewer 2: ```suggestion
If you want to pass attributes to the actual html element that the form field renders, you can add 'htmlAttributes' to the options object. This object can contain things like placeholder, name of maxlength. You can read the ember-paper documentation for more information.
```

Maybe add a link to the relevant section of the ember-paper documentation

## Available types

The following paper form field types are available.

### number-field

> Renders a HTML-input[type="text"], wrapped by a component with attributes, event handlers and build-in methods. You can overwrite them by passing attributes/methods as options, as described above.

Reviewer 2: ```suggestion
Renders an HTML input[type="text"] element, wrapped by a component with attributes, event handlers and build-in methods. You can overwrite them by passing attributes/methods as options, as described above. Possible attributes and methods for this field include:
```

`{{paper-input type="number"}}` renders the input with `type="number"` does our component not do this?

**decimals** By how many decimals your number should be rounded. Default is 2.

**keyDown(e)** By default you can use arrow-up and arrow-down to increase/decrease the number by one. It _won't_ round the number, so it's useable with floats.

> **formatOnChange(value)** Gets the value and formats its, before updating the object. By default it formats a number to '17 009,87'.

Reviewer 2: ```suggestion
**formatOnChange(value)** Gets the value and formats its, before updating the object. By default it formats a number with a space as thousands separator and a comma as decimal separator, like so: '17 009,87'.
```

#### Example

```javascript
options: {
  decimals: 0
}

htmlAttributes: {
  minValue: 0
  maxValue: 99
}

```

> ```handlebars

Reviewer: I guess you could just do `hbs`

{{paper-form-field type="number-field"

>                    object="bakery"

Reviewer: Object would never be a string?

                   propertyPath="baguettes"
                   options=options

>                    htmlAttributes=htmlAttributes

Reviewer 2: So, this example shows the `htmlAttributes` passed directly to the component, while the text above describes adding them to the `options` attribute. Which method is correct?
